frontend/src/pages/Homepage.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components/macro'
+import {Link} from "react-router-dom";
+
+export default function Homepage() {
+    return (
+        <HomeLayout>
+            <h1>Teddys Corona Diaries</h1>
+            <HomeButton to="/recipes">REZEPTE</HomeButton>
+            <HomeButton to="/coronadetails">AKTUELLE LAGE</HomeButton>
+            <HomeButton to="/">LOGIN</HomeButton>
+            <ExternalLink href="https://drive.google.com/drive/folders/16O8ecfl5gzHCjY4jLVWMUXsUTDzN84dm?usp=sharing" target="_blank">HOL DIR DAS EBOOK</ExternalLink>
+        </HomeLayout>
+    )
+}
+
+const HomeLayout = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(https://storage.googleapis.com/teddys-corona-diaries-images/teddy-frontcover.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 100vh;
+  gap: 20px;
+
+  h1 {
+    font-family: Space Mono;
+    margin-top: 50px;
+    font-size: 40px;
+    font-weight: lighter;
+    @media (min-width: 550px) {
+      font-size: 40px;
+    }
+    @media (max-width: 549px) {
+      font-size: 28px;
+    }
+  }
+`
+
+const buttonStyle = `
```

Comment:
you may have to overwrite `:visited` as well, or the color may change for the anchor after it is clicked.